pandas/_libs/window/aggregations.pyx

```diff
@@ -20,15 +20,14 @@ from numpy cimport (
 cnp.import_array()
 
 
-cdef extern from "src/headers/cmath" namespace "std":
+cdef extern from "../src/headers/cmath" namespace "std":
```

Comment:
Without this added `../`, these changes made the build fail for me locally due to not being able to find this file. Not sure why importing from `dtypes` instead of `util` would cause this to happen, but importing from `../` is now consistent with the import on L33